doctor/views.py

```diff
@@ -169,6 +171,28 @@ def make_png_thumbnails_from_range(request) -> HttpResponse:
         return FileResponse(open(filename, "rb"))
 
 
+def xray(request) -> JsonResponse:
+    """Check PDF for bad redactions
+
+    :return: json with bounding boxes and text
+    """
+    form = DocumentForm(request.POST, request.FILES)
+    if not form.is_valid():
+        return JsonResponse(
+            {"error": True, "msg": "Failed validation"}, status=BAD_REQUEST
+        )
+    extension = form.cleaned_data["extension"]
+    if "pdf".casefold() != extension.casefold():
```

Comment:
Well, `casefold` is a new one to me, thanks, but I don't think we need to do it to `"pdf"`. So:

```suggestion
    if "pdf" != extension.casefold():
```

I'm a bit hesitant to use casefold here instead of lower(), since the latter is much more widely known, and I can't imagine an extension that got folded differently than it got lowered (ie., in this use case, I think lower is as good as casefold). But casefold seems like something we should be using more, so I see the value in reaching for it each and every time you have the need to lower case something.